<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>project_diag_new</class>
 <widget class="QDialog" name="project_diag_new">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>409</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create New Project</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item row="1" column="3" rowspan="6">
    <widget class="QPlainTextEdit" name="text_description">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>4</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Type:</string>
     </property>
     <property name="buddy">
      <cstring>combo_ptype</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="combo_ptype">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Utility</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Application</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Library</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Protocol</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>18</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>line_name</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_9">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Description:</string>
     </property>
     <property name="buddy">
      <cstring>text_description</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="line_name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>&amp;Status:</string>
     </property>
     <property name="buddy">
      <cstring>combo_status</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="combo_status">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Spark</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Plan</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Design</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Implement</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Test</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Document</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Release</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Iterate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Maintain</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Decommission</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Dead</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Category:</string>
     </property>
     <property name="buddy">
      <cstring>combo_category</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="combo_category">
     <item>
      <property name="text">
       <string>Development</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Internet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Accessory</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Media</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Game</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Office</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Education</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Science</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Grapics</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>System</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Costomization</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Priority:</string>
     </property>
     <property name="buddy">
      <cstring>spin_priority</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QSpinBox" name="spin_priority">
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>C&amp;hallenge:</string>
     </property>
     <property name="buddy">
      <cstring>spin_challenge</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QSpinBox" name="spin_challenge">
     <property name="maximum">
      <number>9</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_name</tabstop>
  <tabstop>combo_ptype</tabstop>
  <tabstop>combo_status</tabstop>
  <tabstop>combo_category</tabstop>
  <tabstop>spin_priority</tabstop>
  <tabstop>spin_challenge</tabstop>
  <tabstop>text_description</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>project_diag_new</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>project_diag_new</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
